apps/src/StudioApp.js

```diff
@@ -488,6 +488,10 @@ StudioApp.prototype.init = function (config) {
     );
   }
 
+  // Trigger a resize event on init to ensure that the visualization is sized correctly and prevent
+  // video thumbnails from being stretched over visualization column.
+  this.resizeVisualization();
```

Comment:
Just to be safe, do you mind breaking this out into a separate change, as this will affect all labs? The change seems fine to me, but just in case we need to revert

Reply:
nice point, thanks for noticing! Done: https://github.com/code-dot-org/code-dot-org/pull/54994